app/main/download_data.py

```diff
@@ -14,7 +14,7 @@ def quarter_to_date(quarter, year):
         "1": f"{start_year}-04-01T00:00:00Z",
```

Comment:
Also, It all feels a bit wrong this (looking at the useage in app/main/routes.py) since it returns the **start** date of the `reporting_end_period ` but surely this param should signify the end of the quarter period?

Reply:
> Also, It all feels a bit wrong this (looking at the useage in app/main/routes.py) since it returns the **start** date of the `reporting_end_period ` but surely this param should signify the end of the quarter period?

I'm actually not sure now I think about it, is the user expecting the reporting_end_period to be inclusive or exclusive of the quarter they selected? I will see if any of the user researchers can clarify this.

Reply:
@bobbi-hbrown think this has been clarified now, so _To_  date should be the end of the financial quarter in question.